meeting-notes/20181127.md

```diff
@@ -0,0 +1,227 @@
+# Dynamic Data and Capabilities WG — November 27th, 2018
+
+- **Lead:** @pgte
+- **Notetaker:** @jimpick
+- **Attendees:**
+  - @pgte
+  - @jimpick
+  - @andreforsousa
+  - @daviddahl
+  - @arkadiy
+  - @aschmahmann
+  - @marcooliveira
+  - @satazor
+
+- [**Zoom.us meeting URL**](https://protocol.zoom.us/j/586782711)
+- [**Recording**](https://www.youtube.com/watch?v=CY_cVPEATc8&feature=youtu.be)
+
+## Agenda
+
+1. Start recording
+1. Ask everyone to put their name into the list of attendees
+1. Ask for a volunteer to take notes
+1. Everyone can add items to this agenda for things they would like to discuss
+1. Round of intros and updates
+  - Introduce yourself and your interest in this subject
+  - What have you accomplished since the last meeting?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+1. Demos
+  - peer-pad E2E load tests
+  - discussify replies
+1. Questions
+
+### 2 Week Updates:
+
+(please copy this and add your update here :)
+
+#### @
+ - Concluded:
+ - In progress:
+ - Blocked:
+ - Next:
+
+#### @satazor
+ - Concluded:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - In progress:
+   - versidag + ipfs-versidag modules
+     - https://github.com/ipfs/dynamic-data-and-capabilities/issues/50
+ - Blocked:
+   - Add persistance of the data (pinner + sync UI language)
+     - Infrastructure
+     - Sync UI language being done for peerpad; will later borrow the same UI language
+ - Next:
+   - Integrate ipfs-versidag into discussify so that we have history
+     - This will require a small refactor since some of stuff currently in the CRDT is no longer necessary
+   - Either launch discussify in chrome/firefox store
+     - Prepare manifest.json for Firefox so that it's compatible
+     - Add versioning + TTL to the data model
+     - Implement new header (smaller and prettier)
+     - Publish on store: logo + copy + public key?
+   - ...or kickstart Identity/IDM
+   - Prepare talk about DApps + IPFS + peer-star-app for OPO.js
```

Comment:
Will Discussify be released on the app store for OPO.js talk?

Reply:
I think it's possible, even if the history/new header aren't ready. We can launch those features later on via an update. @marcooliveira perhaps the launch plan should have this in mind